docs/guides/composer.mdx

```diff
@@ -0,0 +1,731 @@
+
+   Learn more in the [Visualizations](#visualizations) section.
+
+## Build, edit, and inspect quantum circuits
+
+<Admonition type="warning" title="Download your circuit files before exiting the Composer">
+
+If you want to continue working with a circuit at a later time, be sure to download your circuit file and store it locally before exiting your current Composer session. Use the Save file link in the upper-right corner, or go to the File menu and choose Save file. When you are ready to work with the circuit again, go to the File menu and choose Upload .qasm file, then navigate to your circuit file on your local drive and click Open.
+</Admonition>
+
+### 1. Open IBM Quantum Composer
+
+1. (Optional) If you are not currently signed in to IBM Quantum, select **Sign in** in the upper right corner. Then, you can either sign in or **Create an IBM Cloud account**.
+
+<Admonition type="note">
+   If you don’t sign in, the visualizations automatically show simulated results for up to four qubits. If you want to run your circuit on a quantum computer, or if you want to visualize a circuit that has more than four qubits, you must sign in.
+</Admonition>
+
+2. Open IBM Quantum Composer by clicking the link on the [Learning page](/learning) navigation. The workspace displays an untitled empty circuit. You can either create a new circuit, or upload a .qasm file to continue working with with a circuit you've already created. 
+
+3. Name your circuit by clicking on the words **Untitled circuit** and typing in a name for your circuit. Click the checkmark to save the name.
+
+4. (Optional) Customize your workspace:
+   * Use the **View** menu to change from the default theme to a monochrome theme. You can also select which panels to include on your workspace, then use the menu in the right corner of any panel to access options for further customization. The options to show or hide phase disks, choose the alignment of the qubits on your circuit, and reset the workspace to the default are in the View menu as well.
+   * Switch between dark and light workspace themes in the bottom-right corner of the footer.
+
+To build a circuit, you can either [drag and drop operations](#2-build-your-circuit-with-drag-and-drop), or you can enter [OpenQASM code](#3-build-your-circuit-with-openqasm-code) into the code editor.
+
+### 2. Build your circuit with drag-and-drop
+
+#### Operations catalog
+
+Drag and drop operations from the operations catalog onto the quantum and classical registers. Click the search icon and enter a term in the search bar to quickly find an operation.
+
+Collapse and expand the operations catalog by clicking the icon in the upper right corner of the operations panel. Click the icon next to it to toggle between a grid and list view of the catalog.
+
+Right-click an operation icon and select Info to view the definition of an operation, along with its QASM reference.
```

Comment:
Does IBM want us to call it right-click or secondary-click? Fwiw, Apple calls it a secondary click in their UI, I think.

Reply:
IBM Style uses "right-click".